app/src/main/java/com/mobile/tandil/kotlinbaseproject/fragment/DatePickerDialogFragment.kt

```diff
@@ -0,0 +1,30 @@
+package com.mobile.tandil.kotlinbaseproject.fragment
+
+import android.app.DatePickerDialog
+import android.content.Context
+import android.os.Bundle
+import android.widget.DatePicker
+import androidx.fragment.app.DialogFragment
+import java.util.Calendar
+
+class DatePickerDialogFragment(val listener: (date: String) -> Unit) :
```

Comment:
do not send a listener by parameter, you should use a bundle

Reply:
done